<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RinexConvertSettingDialog</class>
 <widget class="QDialog" name="RinexConvertSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rinex Conversion Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rinex Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MyComboBox" name="rinexVersionComboBox"/>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Interval[/s]:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time Period Separated File:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QRadioButton" name="sepFileTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="sepFileFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Conversion Period:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="MyDateTimeEdit" name="startDateTimeEdit">
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyDateTimeEdit" name="endDateTimeEdit">
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>GPS:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QRadioButton" name="gpsTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QRadioButton" name="gpsFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="4">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>BDS:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QRadioButton" name="bdsTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QRadioButton" name="bdsFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="4" rowspan="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" rowspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>GLO:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" rowspan="2" colspan="2">
      <widget class="QRadioButton" name="gloTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="3" rowspan="2">
      <widget class="QRadioButton" name="gloFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="4" rowspan="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>GAL:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" rowspan="2" colspan="2">
      <widget class="QRadioButton" name="galTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item row="8" column="3" rowspan="2">
      <widget class="QRadioButton" name="galFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item row="9" column="4" rowspan="2">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0" rowspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>QZSS:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" rowspan="2" colspan="2">
      <widget class="QRadioButton" name="qzssTrueRadioButton">
       <property name="text">
        <string>True</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item row="10" column="3" rowspan="2">
      <widget class="QRadioButton" name="qzssFalseRadioButton">
       <property name="text">
        <string>False</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item row="11" column="4">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="MyNoWheelSpinBox" name="intervalSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Components/MyComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyNoWheelSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">Components/MyNoWheelSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header location="global">Components/MyDateTimeEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
